example/views/index.jsx

```diff
@@ -0,0 +1,11 @@
+// eslint-disable-next-line no-unused-vars
```

Comment:
added the necessary options to the example project, but packaging sounds like a good thing. But it may be rather a thing like complate-jsx, right?